db/migrate/20230921222007_create_post.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :author_id, null: false, foreign_key: { to_table: :users }
```

Comment:
- [ ] It is important the column name in the post table be `author_id`, however, because of the `t.references:author_id`, the column name is `author_id_id`. To fix it we should rename the `t.references:author_id` to `t.references:author` instead
![Screenshot from 2023-09-22 00-19-50](https://github.com/Dachrono/blog-app/assets/46388113/954df207-4a3a-4a83-a150-3393954f2470)